packages/grid/x-grid/src/tests/export.XGrid.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { GridApiRef, useGridApiRef, XGrid } from '@material-ui/x-grid';
+import { expect } from 'chai';
+import * as React from 'react';
+import { createClientRenderStrictMode } from 'test/utils';
+
+describe('<XGrid /> - apiRef', () => {
```

Comment:
The correct context:

```suggestion
describe('<XGrid /> - Export', () => {
```